src/array_util.c

```diff
@@ -1912,8 +1912,9 @@ static inline bool _avx2_memequals(const void *s1, const void *s2, size_t n) {
     }
 
     while (ptr1 < end8) {
```

Comment:
how about replace the last two while-loop with a `memcmp`?

Reply:
Could work but I don’t think it would be faster.